### DISCUSSION TITLE
[Angular] rewind is missing in swiper-options.d.ts

### DISCUSSION THREAD

**Discussion by FCHAPLIN** [nolimits4web/swiper#6174] [Q&A] [answered]
Hi ! 

I wanted to use rewind parameter with swiper Angular. 
But the parameter is missing in swiper-options.d.ts... 
Before creating an issue, I just wanted to confirm this is a bug. 

What do you think ?

**Comment by vltansky** [comment, marked as answer]
Are you using the latest version?
<img width="864" alt="image" src="https://user-images.githubusercontent.com/5851280/198661473-d12ad8fc-065c-45ec-8ba3-f65cda78c546.png">

**Reply by vltansky** [reply]
https://unpkg.com/browse/swiper@8.4.4/types/swiper-options.d.ts

**Reply by FCHAPLIN** [reply]
Obviously not ! Thank you!
